ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh

```diff
@@ -51,6 +51,8 @@ rm -f ${SHARED_DIR}/index.json
 folder_name=$(ls -t -d /tmp/*/ | head -1)
 jq ".iterations = $ITERATIONS" $folder_name/index_data.json >> ${SHARED_DIR}/index_data.json
 
+cp ${SHARED_DIR}/index_data.json cp ${SHARED_DIR}/${WORKLOAD}-index_data.json 
```

Comment:
Can we take this change in a separate PR please? Because in this PR it is doing nothing but just creating additional files which are not being used anywhere.

Reply:
this is a building block as part of netobserv performance pipeline since we're aiming to have multiple workloads run in parallel so having workload specific index-data json is needed there.

Reply:
okay, looks fine to me. Tagging @qiliRedHat @skordas to be looped in about this change. Once we fully apply this to all our workloads, we need to remove `index_data.json` from the shared directory and use `${WORKLOAD}-index_data.json` in QE jenkins pipelines to avoid redundant files with same data.

Reply:
@vishnuchalla indeed, we should remove duplicate ones. Btw, I updated the code to include index_data.json as artifact from the step.